integration/docker_engine_test.go

```diff
@@ -294,39 +288,35 @@ func TestRun_WithDockerEngine(t *testing.T) {
 				assert.NoError(t, err, "failed to write .trivyignore")
 				defer os.Remove(trivyIgnore)
 			}
-			if !tc.invalidImage {
-				trivyArgs = append(trivyArgs, "--output", of.Name())
-			}
 			trivyArgs = append(trivyArgs, tc.testfile)
 
 			err = app.Run(trivyArgs)
 			switch {
 			case tc.expectedError != "":
 				require.NotNil(t, err)
 				assert.Contains(t, err.Error(), tc.expectedError, tc.name)
+				return
```

Comment:
Is this needed?

Reply:
Yes, in this case, an error is expected. It means we don't have to compare want and got. The original code is the following.

```
switch {
case tc.expectedError != "":
    require.NotNil(t, err)
    assert.Contains(t, err.Error(), tc.expectedError, tc.name
default:
    ...
}

if !tc.invalidImage {
    // compare want and got
    ...
}
```

So, I've changed it to the following

```
switch {
case tc.expectedError != "":
    require.NotNil(t, err)
    assert.Contains(t, err.Error(), tc.expectedError, tc.name
    return
default:
    ...
}

// compare want and got
compare()
```

I prefer a shallow nest.